packages/nodes-base/nodes/GraphQL/GraphQL.node.ts

```diff
@@ -159,6 +197,214 @@ export class GraphQL implements INodeType {
 				placeholder: 'http://example.com/graphql',
 				description: 'The GraphQL endpoint',
 				required: true,
+				displayOptions: {
+					show: {
+						connectionMode: ['http'],
+					},
+				},
+			},
+			{
+				displayName: 'WebSocket URL',
+				name: 'websocketUrl',
```

Comment:
Could probably re-use the `endpoint` option above

Reply:
Reused, but had to add a more generic URL placeholder to tip the user to us `was://`